style.css

```diff
@@ -3,34 +3,22 @@
     margin: 0;
```

Comment:
- [ ] Use CSS variables: This makes it easier to manage colors or commonly used values throughout the stylesheets.
- [ ] Group Similar Properties: Combine similar selectors into one.
- [ ] Consider Using CSS Preprocessor: If your project is getting bigger, you may consider using a CSS preprocessor like SCSS or LESS, which offers variables, nesting, and other helpful features to write cleaner and more maintainable code.